Multisig/UI/App/RemoteNotificationHandler.swift

```diff
@@ -11,44 +11,31 @@ import UIKit
 import UserNotifications
 import Firebase
 
-extension UNAuthorizationStatus {
-    var hasPermission: Bool {
-        self == .authorized || self == .provisional
-    }
-}
+class RemoteNotificationHandler {
 
-extension UNAuthorizationStatus: CustomStringConvertible {
+    @UserDefault(key: "io.gnosis.multisig.deviceID")
+    private var storedDeviceID: String?
 
-    public var description: String {
-        switch self {
-        case .authorized: return "Authorized (granted)"
-        case .denied: return "Denied"
-        case .notDetermined: return "Not Determined"
-        case .provisional: return "Provisional (granted)"
-        @unknown default: return "Unknown: \(rawValue)"
-        }
-    }
-}
+    @EnumDefault(key: "io.gnosis.multisig.authorizationStatus")
+    private var authorizationStatus: UNAuthorizationStatus?
+
+    @UserDefault(key: "io.gnosis.multiSig.pushToken")
```

Comment:
Same here, I think storing the token in memory is enough right?

Reply:
I would store in the storage because user can close the app and reopen, or something might happen, I thought it would be better to store it.

Reply:
In that case we will receive it again on app start right ?

Reply:
I can't count on Firebase 😄